src/devtools/client/debugger/src/reducers/tabs.js

```diff
@@ -187,7 +187,7 @@ function removeSourcesFromTabList(state, { sources }) {
  * @memberof reducers/tabs
  * @static
  */
-function updateTabList(state, { url, framework = null, sourceId }) {
+function updateTabList(state, { url, sourceId }) {
```

Comment:
Will need to clean up the action as well:
https://github.com/RecordReplay/devtools/blob/2d5a72ebf937567d6fb9dab705b8d2e89c5bb284/src/devtools/client/debugger/src/actions/tabs.js#L17-L28